webpack.config.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devtool: "source-map",
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  externals: {
+    angular: 'angular',
+    lodash: 'lodash'
+  }
+};
```

Comment:
This is just a intermediary module, so there's no need for `webpack` here right?

Reply:
Removed this file